test/rendering/listbox/listbox.js

```diff
@@ -15,6 +15,7 @@
     const app = {
       id: `${+new Date()}`,
       session: {},
+      evaluateEx: async () => (fixture ? fixture.evaluateEx() : null),
```

Comment:
What is this for?

Reply:
For adding the histogram when dimension is not a field but a Master dimension